### DISCUSSION TITLE
How to use /bem/daa method to simulate the plane deformation in 1kgTNT blast in under water?

### DISCUSSION THREAD

**Discussion by tianlu6762** [OpenRadioss/OpenRadioss#3001] [Users]
Sir,
      
How to use /bem/daa method to simulate the plane deformation in 1kgTNT blast in under water?
thanks a lot.

**Comment by tianlu6762** [comment]
How to use /bem/daa method to simulate the plane deformation in 1kgTNT blast in under water?
In thew undex simulation,  standoff point=(0,0,-1000mm), explosion point=(0,0,-10000mm), peak prssure=21.28MPa, theta=0.38566ms.
and plane is located on the water free surface. 


![BEM-DAA](https://github.com/user-attachments/assets/3d221b79-da20-4646-a965-51a0e58ece84)

**Comment by Schwoertzig** [comment]
Hi @tianlu6762 ,
Is your question related to parameter definition or do you ask how to apply it on the targeted sutructure ?

It seems you already defined your parameters, nevertheless you may found additional information about how to determine usual values on user manual page : https://help.altair.com/hwsolvers/rad/topics/solvers/rad/bem_daa_starter_r.htm

Once parameters are defined, this option /BEM/DAA is applied to the targeted discretized surface (See /SURF/SHELL, /SURF/SH3N, or v/SURF/GRBRICK/EXT , or even /SURF/SURF). Surface identifier is then provided to /BEM/DAA (see 'surf_id' parameter on page mentionned above)

Regards

**Comment by tianlu6762** [comment]
Hi Sir,According to the parameter definition of /bem/dda and user manual of Openradioss, i simulate the plane deforestation in undex. But no error message is presented, the job is terminated. I will share my structureal model to you. Would you please take your time to check my model？Best Regards发自我的手机-------- 原始邮件 --------发件人： Thierry Schwoertzig ***@***.***>日期： 2024年9月23日周一 下午4:08收件人： OpenRadioss/OpenRadioss ***@***.***>抄送： tianlu6762 ***@***.***>, Mention ***@***.***>主    题： Re: [OpenRadioss/OpenRadioss] How to use /bem/daa method to simulate the plane deformation in 1kgTNT blast in under water? (Discussion #3001)
Hi @tianlu6762 ,
Is your question related to parameter definition or do you ask how to apply it on the targeted sutructure ?
It seems you already defined your parameters, nevertheless you may found additional information about how to determine usual values on user manual page : https://help.altair.com/hwsolvers/rad/topics/solvers/rad/bem_daa_starter_r.htm
Once parameters are defined, this option /BEM/DAA is applied to the targeted discretized surface (See /SURF/SHELL, /SURF/SH3N, or v/SURF/GRBRICK/EXT , or even /SURF/SURF). Surface identifier is then provided to /BEM/DAA (see 'surf_id' parameter on page mentionned above)
Regards

**Reply by tianlu6762** [reply]
The accessory is my plane model on the undex by using BEM-DAA mothod. 
Would you please take your time to check my model？Best Regards

[DAA-plane.zip](https://github.com/user-attachments/files/17157155/DAA-plane.zip)

**Reply by Schwoertzig** [reply]
Hi @tianlu6762 ,
The algorithmic complexity of /BEM/DAA ( at least O(n²) ) results in high CPU usage, making it a computationally expensive option. However, there is a workaround to optimize and accelerate the Starter process. This involves duplicating the target surface, coarsening it, and tying it with Interface 2. The process is illustrated below. _The limitation mentioned on first capture (~46k) is more of a practical, empirically observed constraint. There is no restriction in the software implementation itself, but the computational cost becomes too high beyond this point_
![image](https://github.com/user-attachments/assets/be0983e3-7ee1-46e9-a709-66a1689c3275)
![image](https://github.com/user-attachments/assets/a71ac938-3e2d-497e-82d5-982fdcc614e6)
![image](https://github.com/user-attachments/assets/91708916-e744-4ed8-a0a1-bb85852872af)
![image](https://github.com/user-attachments/assets/c251ec46-e45f-4ecd-9f2e-66a492755418)
![image](https://github.com/user-attachments/assets/13a60a0e-9a1e-400d-806b-9ba14b93e092)
I hope it will help.
Regards

**Comment by tianlu6762** [comment]
Hi Thierry Schwoertzig，
thanks you very much

**Comment by tianlu6762** [comment]
Hi Thierry Schwoertzig，

Would you please share your model to me?

Best regards.

**Reply by Schwoertzig** [reply]
Hi @tianlu6762 ,
I was only able to find this demonstrator.
![image](https://github.com/user-attachments/assets/223c841c-8db1-4e3b-80d4-5d384b7ef25f)
The input file is attached.
[demonstrator.zip](https://github.com/user-attachments/files/17185539/demonstrator.zip)
Regards

**Comment by tianlu6762** [comment]
Hi Sir,I owe you.You are a nice man.Good luck for youRegards.发自我的手机-------- 原始邮件 --------发件人： Thierry Schwoertzig ***@***.***>日期： 2024年9月30日周一 下午3:10收件人： OpenRadioss/OpenRadioss ***@***.***>抄送： tianlu6762 ***@***.***>, Mention ***@***.***>主    题： Re: [OpenRadioss/OpenRadioss] How to use /bem/daa method to simulate the plane deformation in 1kgTNT blast in under water? (Discussion #3001)
Hi @tianlu6762 ,
I was only able to find this demonstrator.
image.png (view on web)
The input file is attached.
demonstrator.zip
Regards
